.rubocop.yml

```diff
@@ -108,7 +109,7 @@ Style/SpaceInsideHashLiteralBraces:
 Style/StringLiterals:
   Severity: fatal
   Enabled: true
-  EnforcedStyle: double_quotes
+  EnforcedStyle: single_quotes
```

Comment:
I used to use single quotes, as in other languages (and possibly ruby) single quotes were quicker to process, as they were simple strings.

Will this allow both?